### DISCUSSION TITLE
Black Screen during Ventura installation

### DISCUSSION THREAD

**Discussion by nicholasboari** [sileshn/Ryzentosh#37] [Q&A]
I was able to use normal Monterey, but with Ventura the installation stops after 1 hour and 20 minutes and a black screen appears :C

**Comment by sileshn** [comment]
Need information on your pc specs.

**Reply by nicholasboari** [reply]
Ryzen 5 5600
B450m DS3H V2
RX 570 4gb

I used a recovery image, did you use an offline image?

**Comment by sileshn** [comment]
It should work. I always use offline image
